<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DEncrypt21+</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Images.qrc">
    <normaloff>:/Images/images/as.png</normaloff>:/Images/images/as.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>721</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Freeze</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>401</width>
        <height>351</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans';&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans'; color:#ffffff;&quot;&gt;    &lt;/span&gt;&lt;span style=&quot; font-family:'Sans'; color:#000000;&quot;&gt;Files to be frozen:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans'; color:#ffffff;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>455</x>
        <y>30</y>
        <width>221</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Password</string>
      </property>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>60</y>
         <width>191</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Show password</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>30</y>
         <width>191</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>455</x>
        <y>135</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Merge and compile files</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Unfreeze</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>401</width>
        <height>351</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans';&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans'; color:#000000;&quot;&gt;    Solved files:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans'; color:#ffffff;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>455</x>
        <y>30</y>
        <width>221</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Password</string>
      </property>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>60</y>
         <width>191</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Show password</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>30</y>
         <width>191</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>130</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Unfreeze file</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>130</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Export all data</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_File_rcc"/>
   <addaction name="actionClose_New_File_rcc"/>
   <addaction name="separator"/>
   <addaction name="actionOpen_File_rcc"/>
   <addaction name="actionClose_File_rcc"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>771</width>
     <height>22</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>Unfreeze</string>
    </property>
    <addaction name="actionOpen_File_rcc"/>
    <addaction name="actionClose_File_rcc"/>
   </widget>
   <widget class="QMenu" name="menuNew">
    <property name="title">
     <string>Freeze </string>
    </property>
    <addaction name="actionNew_File_rcc"/>
    <addaction name="actionClose_New_File_rcc"/>
   </widget>
   <addaction name="menuNew"/>
   <addaction name="menuOpen"/>
  </widget>
  <action name="actionOpen_File_rcc">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/Images/images/rcc.png</normaloff>:/Images/images/rcc.png</iconset>
   </property>
   <property name="text">
    <string>Open File (.rcc)</string>
   </property>
  </action>
  <action name="actionClose_File_rcc">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/Images/images/rcc_close.png</normaloff>:/Images/images/rcc_close.png</iconset>
   </property>
   <property name="text">
    <string>Close File (.rcc)</string>
   </property>
  </action>
  <action name="actionNew_File_rcc">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/Images/images/new_rcc.png</normaloff>:/Images/images/new_rcc.png</iconset>
   </property>
   <property name="text">
    <string>New File (.rcc)</string>
   </property>
  </action>
  <action name="actionClose_New_File_rcc">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/Images/images/rcc_close.png</normaloff>:/Images/images/rcc_close.png</iconset>
   </property>
   <property name="text">
    <string>Close New File (.rcc)</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
